s3-lambda-translate-cdk-python/requirements.txt

```diff
@@ -0,0 +1,7 @@
+boto3>=1.28.57
+awscli>=1.29.57
+botocore>=1.31.57
+aws-cdk-lib==2.121.1
+constructs>=10.0.0,<11.0.0
+aws-cdk.aws-apigatewayv2-alpha
```

Comment:
I don't think you need the APIGW requirements. I don't see APIGW used in this solution.

Reply:
Done.